Euclidean_Space.R

```diff
@@ -0,0 +1,83 @@
+#Euclidean_Space
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+EuclideanSpace <- setRefClass("EuclideanSpace",
+                              contains = Manifold,
+                              methods = list(
+
+                                Init <- function(self, dimension){
+                                  stopifnot(dimension %% 1 == 0, dimension > 0)
+                                  self$dimension <- dimension
+                                  # self$metric <-
+                                },
+
+                                Belongs <- function(self,point){
+                                  # Evaluate if a point belongs to the Euclidean space.
+                                  point <- gs$backend$to_ndarray(point, to_ndim = 2)
+                                  n_points = point$shape
+                                  points_dim = point$shape
+                                  belongs = point_dim == self$dimension
+                                  belongs = gs$backend$repeat(belongs, repeats = n_points, axis = 0)
+                                  belongs = gs$backend$to_ndarray(belongs, to_ndim = 2, axis = 1)
+
+                                  return(belongs)
+                                },
+
+                                RandomUniform <- function(self, n_samples = 1){
+                                  #Sample in the Euclidean space with the uniform distribution.

```

Comment:
This commented line, which is the "documentation" of the function RandomUniform, should be written in the ReferenceClass style, which is the python style, i.e.:
"Sample in the Euclidean space with the uniform distribution".

See paragraph 'RC' in:
http://r-pkgs.had.co.nz/man.html#man-classes

And last paragraph of section "Writing Reference Methods" in 👍 https://www.rdocumentation.org/packages/methods/versions/3.5.1/topics/ReferenceClasses